README.md

```diff
@@ -7,6 +7,85 @@ Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
 
 ## Introduction
 
+mod-settings is a key-value store. It is an attempt to
```

Comment:
How about this?

> mod-settings is a key-value store that provides a more secure configuration system than the old mod-configuration.

Less mired in history.

Reply:
Good. Thx.